packages/core/src/api/branding.ts

```diff
@@ -0,0 +1,51 @@
+/**
+ * Copyright (c) 2024, WSO2 LLC. (https://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import {AuthClient} from '../auth-client';
+import AsgardeoUIException from '../exception';
+import {BrandingPreferenceAPIResponseInterface, BrandingPreferenceTypes} from '../models/branding-api-response';
+
+/**
+ * Fetch branding preferences from the server.
+ *
+ * @returns {Promise<BrandingPreferenceAPIResponseInterface>} A promise that resolves with the branding preferences.
+ * @throws {AsgardeoUIException} If an error occurs while fetching the branding preferences or when the response is unsuccessful.
+ */
+const branding = async (): Promise<BrandingPreferenceAPIResponseInterface> => {
+  const {
+    baseUrl,
+    type = BrandingPreferenceTypes.Org,
+    name = 'WSO2',
```

Comment:
Is this a fallback?

Reply:
Yes
since they are optional properties in the config, added these as a fallback.